msgraph/models.go

```diff
@@ -540,6 +540,22 @@ type BaseNamedLocation struct {
 	ModifiedDateTime *time.Time  `json:"modifiedDateTime,omitempty"`
 }
 
+type ClaimsMappingPolicy struct {
+	ODataId               *odata.Id `json:"@odata.id,omitempty"`
```

Comment:
Lastly, instead of defining this field here, it's better to embed the `DirectoryObject` struct unless it doesn't make sense to. e.g.

```suggestion
	DirectoryObject

```

Reply:
Got it. Thank you.